packages/base/card-api.gts

```diff
@@ -2035,10 +2039,20 @@ export type FieldDefConstructor = typeof FieldDef;
 export function subscribeToChanges(
   fieldOrCard: BaseDef | BaseDef[],
   subscriber: CardChangeSubscriber,
+  enclosing?: { fieldOrCard: BaseDef; fieldName: string },
```

Comment:
Is it reasonable to say that the subscriberConsumer values only applies to contain compound fields. Or is there an instance that enclosing is actually a "card"

Reply:
it's nested. so at the outer most layer its always a card, and then you have an entry for each layer of compound field that you go thru